recipes/click_config_file/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "click_config_file" %}
+{% set version = "0.4.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 12bc380aa39734749a070b498a21d8d0b2314e38f9320b592fe213a316be5354
+
+build:
+  number: 0
```

This looks like it could be a noarch package?
```suggestion
  number: 0
  noarch: python
```